docs/generate_genome_data.md

```diff
@@ -11,7 +11,7 @@ The scripts used in the process are located in the `src/loaders` directory.
 1. Retrieve Source Genome Files
     * From NCBI
```

Comment:
```suggestion
    * From GTDB
```

Reply:
Why GTDB? we download files from the NCBI FTP server though.

Reply:
Because the collection we're creating is GTDB, and we're downloading the NCBI IDs that occur in that collection from GTDB.

Reply:
Updated!

Reply:
:+1: